subprojects/docs/src/docs/userguide/migration/upgrading_version_8.adoc

```diff
@@ -212,6 +212,78 @@ Some of them still registers conventions for backwards compatibility.
 Registering conventions does not emit a deprecation warning yet to provide a migration window.
 Future Gradle versions will do.
 
+[[project_report_convention_deprecation]]
```

Comment:
Please look at what I merged in for the Java Plugin and the Application Plugin and make this part a bit more consistent with that.